google-cloud-pub-sub-grpc/src/main/scala/akka/stream/alpakka/googlecloud/pubsub/grpc/javadsl/GooglePubSubExternal.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.googlecloud.pubsub.grpc.javadsl
+
+import java.time.Duration
+import java.util.concurrent.{CompletableFuture, CompletionStage}
+
+import akka.actor.Cancellable
+import akka.stream.javadsl.{Flow, Keep, Sink, Source}
+import akka.{Done, NotUsed}
+import com.google.pubsub.v1._
+
+object GooglePubSubExternal {
+
+  def publish(parallelism: Int, publisher: GrpcPublisher): Flow[PublishRequest, PublishResponse, NotUsed] =
+    Flow
+      .create[PublishRequest]
+      .mapAsyncUnordered(parallelism, publisher.client.publish)
+
+  def subscribe(request: StreamingPullRequest,
+                pollInterval: Duration,
+                subscriber: GrpcSubscriber): Source[ReceivedMessage, CompletableFuture[Cancellable]] = {
+    val cancellable = new CompletableFuture[Cancellable]()
+
+    val subsequentRequest = request.toBuilder
+      .setSubscription("")
+      .setStreamAckDeadlineSeconds(0)
+      .build()
+
+    subscriber.client
+      .streamingPull(
+        Source
+          .single(request)
+          .concat(
+            Source
+              .tick(Duration.ZERO, pollInterval, ())
```

Comment:
Shouldn't this be `pollInterval` as initial delay, as well?

Reply:
Yup. Nice catch.